name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -762,6 +762,14 @@ DivvyDiaryMsgUploading = Uploading holdings to DivvyDiary
 
 DivvyDiaryUploadSuccessfulMsg = Completed upload to DivvyDiary successfully
 
+MyDividends24ConfirmUpload = Caution: Importing will overwrite your existing Depot on myDividends24!
```

Comment:
An easier solution of the analogous same labels would be if you use the translations from the DivvyDiary labels and replace the domain.